models/movies_genres.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable indent */
```

Comment:
Any time you have to disable eslint it means you've done something wrong. I suspect you have your tabs set to 4 spaces instead of 2.